lib/rack/common_logger.rb

```diff
@@ -66,8 +66,8 @@ def log(env, status, header, began_at)
     end
 
     def extract_content_length(headers)
-      value = headers[CONTENT_LENGTH] or return '-'
-      value.to_s == '0' ? '-' : value
+      value = headers[CONTENT_LENGTH]
+      value.nil? || value.to_s == '0' ? '-' : value
```

Comment:
While very unlikely to happen, this also changes the behavior when `headers[CONTENT_LENGTH]` is `false`.

```ruby
value = headers[CONTENT_LENGTH]
!value || value.to_s == '0' ? '-' : value
```

to keep the same behavior.

Reply:
I think the value from `headers[CONTENT_LENGTH]` can be a `String` or `nil`. Is it possible to be true or false ?

Reply:
Like this warning "a header value must be a String, but the value of 'Content-Length' is a FalseClass"